test/run-tests

```diff
@@ -353,6 +354,42 @@ class Task:
                 f"in image {self.image['name']}"
             )
 
+    def is_role_distro_supported(self):
+        """
+        Return True if image is supported. Otherwise False.
+        """
+        distro = self.image["name"].split("-")[0]
+        version = self.image["name"].split("-")[1]
+        if version == "x":
+            image_filename = self.get_url().rsplit("/", 1)[1]
```

Comment:
Please use the same method as https://github.com/linux-system-roles/test-harness/blob/3796e3f92d2c71fc74e2d1889edc2ad82966e415/test/run-tests#L257